src/collections/posts/custom-songs.md

```diff
@@ -2,20 +2,20 @@
 title: Getting Started with Custom Songs
 section: speciality
 category: guide
-publish: 2024-03-22T00:00:00.000Z
+publish: 2024-06-15T00:00:00.000Z
```

Comment:
Generally speaking: Maybe we want a last updated field?

Reply:
Since we don't pull it from a backend but from a file and the way the CMS works it would be entirely user-defined. So I don't see an issue with that. The otherway around would be hard case. Maybe candidate for a ticket then?

Reply:
Sure! We're filling up with tickets! 😬